src/APBFramelessWindow/NativeTranslucentFramelessWindow.cpp

```diff
@@ -43,15 +42,12 @@ NativeTranslucentFramelessWindow::NativeTranslucentFramelessWindow(QWidget *pare
 bool NativeTranslucentFramelessWindow::nativeEvent(const QByteArray &eventType, void *message, long *result) {
 	MSG *nativeMessage = static_cast<MSG *>(message);
 	LPARAM lparam = nativeMessage->lParam;
-	// WPARAM wparam = nativeMessage->wParam;
-	// HWND hwnd = nativeMessage->hwnd;
 	UINT msg = nativeMessage->message;
 
 	switch(msg) {
 		case WM_NCCALCSIZE: {
 			// Abort computing & drawing of the non client area.
 			*result = 0;
-
```

Comment:
Clean code principle: `Use white space to associate related things and disassociate weakly related.`